docs/sources/dashboards/assess-dashboard-usage/index.md

```diff
@@ -25,7 +25,7 @@ weight: 200
 Usage insights enables you to have a better understanding of how your Grafana instance is used.
 
 {{% admonition type="note" %}}
-Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud/). Grafana Cloud insights logs include additional fields with their own dashboards. Read more in the [Grafana Cloud documentation](/docs/grafana-cloud/usage-insights/).
+Available in [Grafana Enterprise][] and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud/). Grafana Cloud insights logs include additional fields with their own dashboards. Read more in the [Grafana Cloud documentation](/docs/grafana-cloud/usage-insights/).
```

Comment:
I'm gonna have to change this one because reference style links don't work inside admonition shortcodes.

How important is the link to [Grafana Enterprise](https://grafana.com/docs/grafana/latest/introduction/grafana-enterprise/) in this note?

Reply:
Is it possible to use `/docs/grafana/introduction/grafana-enterprise/` for this? Because this is pretty standard text throughout docs, I'm hesitant to remove the Enterprise links entirely. But if there isn't a clean way to do it, then I think we take it out in these spots.

Reply:
Yeah, I think I'm gonna have to see if I can find a way. The need to link from admonitions is too strong for it to be something we can't do.